test_issue_maker.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+import re
+from flask import Flask, request
+from werkzeug import ImmutableMultiDict
+from swaglyrics import __version__
+from issue_maker import update, genius_stripper, check_song, add_stripper
+from issue_maker import passwd
+from issue_maker import del_line
+from unittest.mock import patch
+from app import app
+
+
+class FlaskTestCase(unittest.TestCase):
+    def setUp(self):
+        self.song = '거품 안 넘치게 따라줘 [Life Is Good] (feat. Crush, Dj Friz)'
+        self.artist = 'Dynamic Duo'
+        self.correct_title = f'{self.song} by {self.artist}'
+        self.path = 'I don\'t know'
+
+    def tearDown(self):
+        pass
+
+    def test_update(self):
+        app = Flask(__name__)
+        with app.test_request_context('/'):
+            """Test for correct return if version != request.form['version']"""
+            request.form = ImmutableMultiDict([('version', str(__version__) + 'bla')])  # faulty version
+            request.form = ImmutableMultiDict([('song', 'Sweeter Without You')])  # song in unsupported.txt
```

Comment:
Is it possible to mock unsupported songs list or something? I don't want to hardcode values because what if unsupported.txt changes